stdlib/database/sql/src/main/java/org/ballerinalang/database/sql/SQLDatasourceUtils.java

```diff
@@ -995,6 +997,13 @@ public static String getSQLType(BType value) {
             return Constants.SQLDataTypes.BOOLEAN;
         case TypeTags.DECIMAL_TAG:
             return Constants.SQLDataTypes.DECIMAL;
+        case TypeTags.ARRAY_TAG:
+            if ((value instanceof BArrayType) && ((BArrayType) value).getElementType().getTag() == TypeTags.BYTE_TAG) {
```

Comment:
I think we don't need to run instanceof here. If the tag is ARRAY_TAG then it should be an instance of BArrayType.

Reply:
Without this instanceof check i got the following error.
```
Unchecked/unconfirmed cast from org.ballerinalang.model.types.BType to org.ballerinalang.model.types.BArrayType in org.ballerinalang.database.sql.SQLDatasourceUtils.getSQLType(BType) [org.ballerinalang.database.sql.SQLDatasourceUtils] At SQLDatasourceUtils.java:[line 1001] BC_UNCONFIRMED_CAST
```

So, now updated the spotbug rule of sql module with this https://github.com/ballerina-platform/ballerina-lang/pull/15410/commits/cee73a092e226aeb37d10021adfb885100a9e44a
and removed the  instanceof check